docs/fields.md

```diff
@@ -1495,6 +1495,7 @@ NodeStatus contains status information about an individual node in the workflow
 |`inputs`|[`Inputs`](#inputs)|Inputs captures input parameter values and artifact locations supplied to this template invocation|
 |`message`|`string`|A human readable message indicating details about why the node is in this condition.|
 |`name`|`string`|Name is unique name in the node tree used to generate the node ID|
+|`nodeName`|`string`|NodeName is the name of the host on which the workflow is running|
```

Comment:
We have two "nodes" in scope here: an Argo node and a Kubernetes node. I fear that naming this field `nodeName` will overload the definition. Perhaps it should be `hostNodeName` or `runtimeNodeName` or some better name that I probably can't think of.

Do you have any suggestions for a good name that differentiates the two? The descriptions should probably also make this distinction clear.

Reply:
`runtimeHostNodeName`?

Reply:
I like `hostNodeName` alot - will make the change